pkgs/development/python-modules/opensfm/default.nix

```diff
@@ -64,16 +66,25 @@ buildPythonPackage rec {
     echo 'feature_type: SIFT' >> data/berlin/config.yaml
     echo 'feature_type: HAHOG' >> data/lund/config.yaml
 
+    # make opensfm correctly import glog headers
+    substituteInPlace opensfm/src/CMakeLists.txt \
+      --replace \
+        'find_package(glog REQUIRED)' \
+        "add_compile_definitions(GLOG_USE_GLOG_EXPORT)"$'\n'"find_package(glog REQUIRED)"
```

Comment:
Can't we do https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_FLAGS.html#variable:CMAKE_%3CLANG%3E_FLAGS instead? Would be less fragile wrt overrides/updates probably